electron-app/src/magnifier-preload.ts

```
@@ -0,0 +1,44 @@
+  // Receive methods for magnifier-specific updates
+  on: {
+    updatePosition: (
+      callback: (data: {
+        x: number;
+        y: number;
+        displayX: number;
+        displayY: number;
+      }) => void
+    ) => {
+      const subscription = (_event: unknown, data: unknown) =>
+        callback(data as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)